test/recipes/15-test_dhparam.t

```diff
@@ -0,0 +1,97 @@
+#! /usr/bin/env perl
+# Copyright 2017-2020 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+
+use strict;
+use warnings;
+
+use File::Spec;
+use OpenSSL::Glob;
+use OpenSSL::Test qw/:DEFAULT data_file/;
+use OpenSSL::Test::Utils;
+
+setup("test_dhparam");
```

Comment:
We already have a 20-test_dhparam.t so the name of this test is quite confusing.

Reply:
I agree, but can't invent a better one, so name it

Reply:
`20-test_dhparam_check.t` ?

Reply:
Done. Thanks!